Utilities/XrdAdaptor/src/XrdStatistics.cc

```diff
@@ -44,6 +44,26 @@ void XrdStatisticsService::postEndJob()
     }
 }
 
+std::map<std::string, XrdStatisticsService::CondorIOStats>
+XrdStatisticsService::condorUpdate()
+{
+    std::map<std::string, XrdStatisticsService::CondorIOStats> result;
+    XrdSiteStatisticsInformation *instance = XrdSiteStatisticsInformation::getInstance();
+    if (!instance) {return result;}
+
+    std::lock_guard<std::mutex> lock(instance->m_mutex);
```

Comment:
For future: exposing a lock is a poor interface. We should rethink this service.